src/Symfony/Contracts/HttpClient/Test/HttpClientTestCase.php

```diff
@@ -763,4 +763,62 @@ public function testGzipBroken()
         $this->expectException(TransportExceptionInterface::class);
         $response->getContent();
     }
+
+    public function testJsonRegexBuffer(): void
```

Comment:
this should be moved to `HttpClientTestCase` in the `Symfony\Component\HttpClient\Tests` namespace
(as the new behavior is not in the contracts)

Reply:
Moved the test to the correct class